src/test/actions/communityAction.spec.js

```diff
@@ -1,12 +1,13 @@
 import configureMockStore from 'redux-mock-store';
 import thunk from 'redux-thunk';
 import nock from 'nock';
-import * as actions from '../../../store/actions/community';
+import * as actions from '../../store/actions/communityActions';
+import { testBaseUrl } from '../config/testConfig';
 
 const middleware = [thunk];
 const mockStore = configureMockStore(middleware);
 
-describe('test community action', () => {
+describe.only('test community action', () => {
```

Comment:
@chekwas88 I think the describe should not have a `.only` Kindly review it

Reply:
yea.. used it when I was debugging the tests. Thanks for spotting it out.